lib/src/main/java/com/bytehamster/lib/preferencesearch/PreferenceItem.java

```diff
@@ -30,7 +30,10 @@ boolean matchesFuzzy(String keyword) {
         return getScore(keyword) > 0.3;
     }
 
-    boolean matches(String keyword) {
+    boolean matches(String keyword, boolean caseSensitive) {
+        if (!caseSensitive) {
+            return getInfo().toLowerCase().contains(keyword.toLowerCase());
```

Comment:
Oh, really interesting. Thanks, Mike. Fixed it

Reply:
On a second thought, actually isn't it better to use `Locale.getDefault()`, which should match the app's language?

Reply:
@BrayanDSO that's a good point, I think mentally I was stuck on the "stock" normal issue where it is lower-casing fixed strings but in this case they should be translated, so the user's default would be best. However, remember in AnkiDroid that the language can be user selected and *not* the system locale. Tricky one. :thinking: - do we have a Context anywhere we may pull Locale from ? The one may actually require some testing or something to "feel" it and make sure it really works as expected. 

For whatever it is worth, here's the typical issue: https://garygregory.wordpress.com/2015/11/03/java-lowercase-conversion-turkey/ / https://stackoverflow.com/questions/23524231/lower-case-of-turkish-character-dotted-i

Apparently it may be as easy as going toUpperCase instead of lower which has the potential to be a  brilliant "work smarter not harder" solution. I've only ever seen it bite personally on Turkish I when going toLower so maybe it's just that easy.

Reply:
> However, remember in AnkiDroid that the language can be user selected and not the system locale.

`Locale.getDefault()` returns the app language, not system's. I know because of the work on `buildCategorySummary`'s RTL support work (https://github.com/ankidroid/Anki-Android/commit/f9e886fe8ca6eb0f57927c8255205faf9ab704e0#diff-47297e4165b250a242c0ed3e241814b88ca539c48a7f01dedce81a34ee15f3a3R46-R49).

So it should work accordingly with the current approach. 

> Apparently it may be as easy as going toUpperCase instead of lower which has the potential to be a brilliant "work smarter not harder" solution. I've only ever seen it bite personally on Turkish I when going toLower so maybe it's just that easy.

💯

Reply:
> Also, a Turkish can say that “Kedi”.toUpperCase() works on Turkey but won’t work in the rest of the world (Kedi is Cat in Turkish). “Kedi”.toUpperCase() using an en_ES Locale will return KEDI while with Turkish Locale will return KEDİ. It all depends on the perspective

Hmmmm. As per this comment in one of the links you addressed (https://garygregory.wordpress.com/2015/11/03/java-lowercase-conversion-turkey/), vice-versa is valid as well. So uppercasing X lowercasing methods doesn't makes a difference. But as long as locale is kept, it should work

Reply:
"And today I learned..." - but joking tone aside, this is a difficult issue! So I'm glad to learn more, thanks for posting that. Agreed using default locale (assuming that's what "you can pick your language apps" are affecting) seems valid.